Resources/Prototypes/_LostParadise/tags.yml

```
@@ -163,3 +163,7 @@
 
 - type: Tag
   id: LPPProcessedDiamond
+
+- type: Tag
+  id: LPPCandy #Halloween
```

yamllint

[warning] too few spaces before comment

(comments)